### PR TITLE
Update staging-review-processes.md

### PR DESCRIPTION
**Pull request by 1Copenut**
Work related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/16150:
* "Update docs, and share in ToT, so it's clear that VFS teams need to come to the staging review w/their testing and artifacts complete"

### CODE REVIEW

File: platform/accessibility/guidance/staging-review-processes.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 # Accessibility staging review preparation
 
-Teams must conduct their own accessibility testing **before** scheduling a VSP Collaboration Cycle [Staging Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#staging-review). 
+Teams must conduct their own accessibility testing **before** scheduling a VSP Collaboration Cycle [Staging Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#staging-review). VSP accessibility specialists will verify teams have completed foundational accessibility testing before the staging review can start. Teams are **required** to complete the [foundational accessibility tests](#foundational-accessibility-tests-required) and encouraged to complete as many of the [advanced accessibility tests](advanced-accessibility-tests-recommended) as time and techincal proficiency allow.
```
Comment (shiragoodman) [inline platform/accessibility/guidance/staging-review-processes.md:4]: Is advanced-accessibility-tests-recommended linked correctly?

File: platform/accessibility/guidance/staging-review-processes.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+
+If teams have not completed their foundational testing, the VSP accessibility specialist will escalate to DEPO leadership.
```
Comment (shiragoodman) [inline platform/accessibility/guidance/staging-review-processes.md:6]: recommend changing "the VSP accessibility specialist" to just "VSP"

**Review by shiragoodman — CHANGES_REQUESTED** [review]

**Review by peggygannon — APPROVED** [review]
@1Copenut Updates look good to me. Thx!

**Comment by shiragoodman** [comment]
@1Copenut did you see my requested changes?

**Review by shiragoodman — APPROVED** [review]

**Review by rroueche — COMMENTED** [review]
Please change VSP to DEPO Platform, and then good to merge!

**Comment by 1Copenut** [comment]
Done!

**Pull request merged**